.github/workflows/wait-for-checks.yaml

```diff
@@ -17,6 +17,8 @@ jobs:
     permissions:
       checks: read
     steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
```

Comment:
Why are you checking something out?

Reply:
It was a try to get the brew bin but unfortunately it did not have the desired effect